CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Deprecated
     * Deprecated `GroundPrimitive.geometryInstance`. It will be removed in 1.20. Use `GroundPrimitive.geometryInstances` instead.
     * Deprecated `TileMapServiceImageryProvider`. It will be removed in 1.20. Use `createTileMapServiceImageryProvider` instead.
+    * Deprecated `throttleRequests` flag in `TerrainProvider`. It will be removed in 1.19. It is replaced by an optional `Request` object that stores information used to prioritize requests.
```

Comment:
Or maybe `Request` should be public.  Can we reasonably do that without making the entire request scheduler public since I don't think we want to commit to the full public API today.